avocado/utils/asset.py

```diff
@@ -18,6 +18,7 @@
 
 import errno
 import hashlib
+import json
```

Comment:
This type of style change urges to be on its own commit proportionally to the size of the commit itself.  What I mean is, if you're sending a commit with ~10 lines with a fix, it may be OK to have this type of change alongside it, because it won't require much from the reviewer.

But, this commit has ~500 +/- lines:

```
 avocado/utils/asset.py                             | 482 ++++++++++++++++++++++++++++++++++++++++++++++++++-------------------------------------
 selftests/{unit => functional}/test_utils_asset.py |  54 +++++-----
```

So it's a good idea to facilitate the review (and people reading the history in the future) by separating those from the other more important changes.

Reply:
I noticed that after I submitted the PR. Some of the changes will still have a great number of lines based on the code they affect. Anyway, I'll try to improve it in the next version.